### PR TITLE
Add 2.2.1

### PR DESCRIPTION
**Pull request by chneau**
Following PR #78 

Right now, this PR fails the github action tests since node:12.22.2-alpine does not exist yet.

https://github.com/nodejs/docker-node/pull/1515 

Potential work around would be to use `node:12.22-alpine` instead of `node:12.22.2-alpine`. The tests are passing with this work around https://github.com/chneau/meteor-base/actions/runs/1002439653

### CODE REVIEW

**Comment by diavrank** [comment]
12.22.2 now is available in [docker hub](https://hub.docker.com/_/node?tab=tags&page=1&ordering=last_updated&name=12.22.2).

**Review by diavrank — APPROVED** [review]

**Comment by diavrank** [comment]
Seems to be fine. Only missing to wait for @GeoffreyBooth's review.

File: example/app/package.json
Comment (GeoffreyBooth) [inline example/app/package.json:11]: Should this have been downgraded?

Comment (chneau) [reply example/app/package.json:11]: @GeoffreyBooth Hello, this was the result of a `meteor create --release 2.2.1 --blaze app`

`7.12.5` comes from this commit https://github.com/disney/meteor-base/commit/4d4456654eae85b250fec64e217b92e16276336b which is for Meteor 1.12.

Running the command `meteor create --release 1.12 app` gives me a `"@babel/runtime": "^7.11.2",` 

Not sure what is the best move to do here. I can revert this change if you want

Comment (diavrank) [reply example/app/package.json:11]: It had a newer version because I ran `npm update` in order to have the latest versions in the **package.json**. IMHO I always use the latest versions to ensure a project updated and find possible future bugs.

Comment (chneau) [reply example/app/package.json:11]: Should I set it to 7.14.6 ? https://www.npmjs.com/package/@babel/runtime

Comment (diavrank) [reply example/app/package.json:11]: For me it would be ok, but the final decision rely on @GeoffreyBooth .

Comment (GeoffreyBooth) [reply example/app/package.json:11]: I think what we want is for the _tests_ run on the versions of dependencies that are pinned for the particular version of Meteor being tested. But this example app isn’t part of the tests, if I remember correctly. So if that’s the case, then sure, we might as well always have the example use the latest versions as of whenever we update it.

Comment (diavrank) [reply example/app/package.json]: @GeoffreyBooth now it is updated and ready to merge :)

**Pull request merged**
